src/Rules/FunctionCallParametersCheck.php

```diff
@@ -244,6 +246,30 @@ public function check(
 				->build();
 		}
 
+		if (
```

Comment:
it could also make sense to move this code directly into `src/Rules/Functions/CallToFunctionParametersRule.php` 

----

in case you want this error behind a feature-flag we could also create a separate rule. if so, which level?